ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py

```diff
@@ -344,6 +344,45 @@ def test_tensorflow_convert_to_tensor(
     )
 
 
+# cond
+@handle_frontend_test(
+    fn_tree="tensorflow.cond",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        min_num_dims=1,
+        min_dim_size=1,
+    ),
+    pred_cond=st.booleans(),
+    test_with_out=st.just(False),
+)
+def test_tensorflow_cond(
+    *,
+    dtype_and_x,
+    pred_cond,
+    test_flags,
+    on_device,
+    fn_tree,
+    frontend,
+):
+    def _test_true_fn():
+        return list
```

Comment:
The implementation is passing tests, but can you tell me why are you returning `list` in both instances? For example if I change your implementation to `if not pred:`, result will be the same.

Reply:
It's because the docs state that both true_fn and false_fn must return lists of output tensors: https://www.tensorflow.org/api_docs/python/tf/cond#:~:text=true_fn%20and%20false_fn%20both%20return%20lists%20of%20output%20tensors.%20true_fn%20and%20false_fn%20must%20have%20the%20same%20non%2Dzero%20number%20and%20type%20of%20outputs.So I thought the test fn functions could both return lists. However I could be wrong, so I appreciate any other insight on the best way to do it.